Makefile

```diff
@@ -570,7 +570,7 @@ run-docker-dev: ## start a local development Docker container
 		develop
 
 .PHONY: run-hopr-admin
-run-hopr-admin: version=07aec21b
+run-hopr-admin: version=latest
```

Comment:
Pinning an image is always preferable to having a floating version.

Reply:
I agree. Would be nice to have a github workflow to make the update PR once there is a new version of hopr admin